### DISCUSSION TITLE
floorplan take 10 second to load

### DISCUSSION THREAD

**Discussion by boomramada** [ExperienceLovelace/ha-floorplan#377] [Bugs]
I have configured the floorplan as follows:
`
  - title: Heat Map
    path: floorplan
    theme: Google Dark Theme
    panel: true
    type: panel
    badges: []
    cards:
      - config:
          console_log_level: info
          defaults:
            hover_action: hover-info
            tap_action: more-info
          image: /local/custom_ui/floorplan/paradise-320.svg
          cache: true
          rules:
          stylesheet: /local/custom_ui/floorplan/paradise.css
        full_height: true
        type: custom:floorplan-card
`

and according to browser inspect, it takes 10 seconds to load. 

![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/15424398/299f021e-419d-4cca-a9ef-ad8ade1438ac)

Is this normal or bug?

Version : 1.0.39 running on Raspberry Pi4 as a docker.

**Comment by OzGav** [comment]
Firstly we can’t read your yaml because you didn’t format it.

Secondly, browser load times are dependent on factors factors such as your network, your server, your SVG size….you will need to look at all these things.

**Reply by boomramada** [reply]
Sorry, not sure how to get the formatting working here. 
but here is the imaged version of my yaml:

![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/15424398/8b80af02-6827-477b-a35d-f9a10436c0aa)

My SVG is around 130kb (is that ok size?)
I have tried accessing via mobile/home/work/ie and google, they all appear around 10 second mark.

**Comment by boomramada** [comment]
Found the issue.  My CSS  file was empty, I had to add
`
/* SVG size */
`

This resolved the issue, and now it's instant.
